jobs/config.json

```diff
@@ -11477,6 +11477,23 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-multicluster-ingress-test": {
+    "args": [
+      "make",
+      "--",
+      "-j",
+      "5",
+      "test",
+      "coveralls",
+      "vet",
+      "fmt",
+      "lint"
+    ],
+    "scenario": "execute",
```

Comment:
I have no idea what does "execute" mean. Its copied from other existing jobs. Let me know if this is incorrect

Reply:
that's just a bash wrapper - if you don't need to run e2e tests then this is correct

Reply:
yes no e2e test :)